src/modules/UI/Wallets/action.js

```diff
@@ -148,6 +149,7 @@ export const upsertWallets = (wallets: Array<EdgeCurrencyWallet>) => (dispatch:
   if (!loginStatus) {
     dispatch({ type: 'LOGGED_OUT' })
   }
+  dispatch(updateExchangeRates())
   dispatch({
```

Comment:
We will want to re-build the exchange rates every time a wallet is added or changed (as well as setting default fiat changed)